google/generativeai/types/generation_types.py

```diff
@@ -155,6 +156,7 @@ class GenerationConfig:
     top_p: float | None = None
     top_k: int | None = None
     response_mime_type: str | None = None
+    response_schema: Mapping[str, str | None] | None = None
```

Comment:
This needs to get converted to a schema object like we do for FunctionCalling.

Reply:
Actually, maybe the better way of putting it is "this should accept a Mapping or a `glm.Schema`:  `glm.Schema | Mapping[str, str | None] | None`

Reply:
Just updated!